packs/data/feat-effects.db/effect-saoc-astrology.json

```diff
@@ -0,0 +1,141 @@
+{
+    "_id": "6ACbQIpmmemxmoBJ",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>This effect grants you the selected circumstance bonus or penalty to skill checks.</p>\n<hr />\n<p><em>Note: The effect lasts until the end of your turn, and should be manually removed after you have made a skill check.</em></p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 0
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "-1 penalty",
```

Comment:
I think this is way overcomplicated. You can just have the -1,1,2,3,4 with 1, 3,  and 4 predicated for the heir feat. No need for labels here, just value with the number and reference that number with `{item|flags.pf2e.rulesSelections.bonusChoice}` or something.

Reply:
I've simplified this massively - thanks for the advice (at some point, I'll 'get' choicesets properly, but still not quite there!).
I've left the labels in, as when I tested without them, it says I "have no valid options for Effect: blah".

Reply:
Hmm, should be able to just have a value? I'd like to fix that if we can't, instead of having more labels on things.